BUILD.bazel

```diff
@@ -0,0 +1,212 @@
+        "include/albatross/src/evaluation/model_metrics.hpp",
+        "include/albatross/src/evaluation/prediction_metrics.hpp",
+        "include/albatross/src/evaluation/traits.hpp",
+        "include/albatross/src/graph/minimum_spanning_tree.hpp",
+        "include/albatross/src/indexing/apply.hpp",
+        "include/albatross/src/indexing/declarations.hpp",
+        "include/albatross/src/indexing/filter.hpp",
+        "include/albatross/src/indexing/group_by.hpp",
+        "include/albatross/src/indexing/subset.hpp",
+        "include/albatross/src/indexing/traits.hpp",
+        "include/albatross/src/models/conditional_gaussian.hpp",
+        "include/albatross/src/models/gp.hpp",
+        "include/albatross/src/models/least_squares.hpp",
+        "include/albatross/src/models/null_model.hpp",
+        "include/albatross/src/models/patchwork_gp.hpp",
+        "include/albatross/src/models/patchwork_gp_details.hpp",
+        "include/albatross/src/models/ransac.hpp",
+        "include/albatross/src/models/ransac_gp.hpp",
+        "include/albatross/src/models/sparse_gp.hpp",
+        "include/albatross/src/samplers/callbacks.hpp",
+        "include/albatross/src/samplers/ensemble.hpp",
+        "include/albatross/src/samplers/initialization.hpp",
+        "include/albatross/src/samplers/state.hpp",
+        "include/albatross/src/stats/chi_squared.hpp",
+        "include/albatross/src/stats/gauss_legendre.hpp",
+        "include/albatross/src/stats/gaussian.hpp",
+        "include/albatross/src/stats/incomplete_gamma.hpp",
+        "include/albatross/src/stats/ks_test.hpp",
+        "include/albatross/src/tune/finite_difference.hpp",
+        "include/albatross/src/tune/greedy_tuner.hpp",
+        "include/albatross/src/tune/tune.hpp",
+        "include/albatross/src/tune/tuning_metrics.hpp",
+        "include/albatross/src/utils/async_utils.hpp",
+        "include/albatross/src/utils/block_utils.hpp",
+        "include/albatross/src/utils/csv_utils.hpp",
+        "include/albatross/src/utils/eigen_utils.hpp",
+        "include/albatross/src/utils/linalg_utils.hpp",
+        "include/albatross/src/utils/map_utils.hpp",
+        "include/albatross/src/utils/random_utils.hpp",
+        "include/albatross/src/utils/variant_utils.hpp",
+        "include/albatross/src/utils/vector_utils.hpp",
+        "include/albatross/tests/Utils",
+        "include/albatross/utils/AsyncUtils",
+        "include/albatross/utils/CsvUtils",
+        "include/albatross/utils/LinalgUtils",
+        "include/albatross/utils/MapUtils",
+        "include/albatross/utils/RandomUtils",
+        "include/albatross/utils/VariantUtils",
+    ],
+    includes = [
+        "include",
+        "tests",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@cereal",
+        "@eigen",
+        "@fast_csv",
+        "@gzip",
+        "@nlopt",
+        "@variant",
+    ],
+)
+
+swift_cc_test(
+    name = "albatorss-test",
```

Comment:
Let's add the compile options for this target as well: https://github.com/swift-nav/albatross/blob/01e266027fa4096165966f1b6b847b4f11944a2d/tests/CMakeLists.txt#L54